src/Runner.Listener/Runner.cs

```diff
@@ -466,6 +466,7 @@ private void PrintUsage(CommandSettings command)
  --url string     Repository to add the runner to. Required if unattended
  --token string   Registration token. Required if unattended
  --name string    Name of the runner to configure (default {Environment.MachineName ?? "myrunner"})
+ --labels string  Additional labels of the runner (default 'self-hosted', '{Constants.Runner.Platform}', '{Constants.Runner.PlatformArchitecture}')
```

Comment:
should this be `default 'self-hosted,{Constants.Runner.Platform},{Constants.Runner.PlatformArchitecture}'`

Reply:
nm.... this is *additional*

Reply:
`Extra labels in addition to the default: 'self-hosted,{Constants.Runner.Platform},{Constants.Runner.PlatformArchitecture}'`